rpa-adapter/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+  id 'net.ltgt.apt' version '0.14'
+  id 'jacoco'
+}
+
+def springBootVersion = '2.0.1.RELEASE'
+
+def versions = [
+  springBoot: springBootVersion
+]
+
+dependencies {
+  compile project(':domain-model')
+
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter', version: versions.springBoot
```

Comment:
please drop the spring boot version here its not required same in the other mention of this file